hood/crypto_trading/structures.py

```
@@ -1,49 +1,32 @@
+    Union,
+    List,
+    Dict,
+    Generic,
+    TypeVar,
+    TYPE_CHECKING,
+    Optional,
+)
+
+if TYPE_CHECKING:
+    import requests
+    from .. import schema as _schema  # noqa
```

Ruff (0.14.10)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)